### PR TITLE
docs: add a codecov badge (#1062)

### LINKED ISSUE #1062: Add badges to the Readme File for codecov
## Description
Add badges to the Readme File for codecov after this [PR.](https://github.com/anitab-org/vms/pull/983) & update the titile of the Readme file to `Volunteer Management System(VMS)`

## Definition of Done
- [x] Approval by 1 mentor.

## Estimation
1 hours

### PR DESCRIPTION
**Pull request by aniketrb-github**
# Description
- Add a codecov badge (#1062) 
- Updated the readme file title from `AnitaB.org Portal - VMS project` to `Volunteer Management System(VMS)`

Fixes #1062

# Type of Change:
- Documentation

# How has this been tested?
`Before Changes were made:`
before-1062-fix

`After Changes were made:`
after-1062-fix

# Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 ============================
 
-[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
+[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms) ![Build Status](https://codecov.io/gh/anitab-org/portal/branch/develop/graph/badge.svg)
```
Comment (satya7289) [inline README.md:4]: Here, add codecov for the VMS, not for the portal, also add the codecov link to it.

Comment (satya7289) [reply README.md:4]: Add some more description for the PR.

Comment (aniketrb-github) [reply README.md:4]: Thanks for the correction, I will add the codecov Link to it as well.

But regarding the 'not adding it to portal' comment, I have appended the codecov to readme.md file of `vms` project exactly, and not elsewhere.

Comment (satya7289) [reply README.md:4]: No problem, you can update the codecov link. 

Yes right, can you update this `AnitaB.org Portal - VMS project` to `AnitaB.org - Volunteer Management System(VMS) project`.

Comment (aniketrb-github) [reply README.md:4]: @satya7289 just a tiny concern here regarding the project title.
I observed the readme-files of other projects and most of them have project names itself in the readme file, for instance,
`Mentorship System (Backend)` and not `AnitaB.org - Mentorship System (Backend)`
`Mentorship System (Android)`
`Mentorship System (iOS)`
and so on...

So should we keep it in uniform with others which will impart more uniformity? 
If yes then it will be as follows
`Volunteer Management System(VMS)`  instead of AnitaB.org - Volunteer Management System(VMS) project
Please let me know. Thanks.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms) [![codecov](https://codecov.io/gh/anitab-org/mentorship-backend/branch/develop/graph/badge.svg)](https://codecov.io/gh/anitab-org/mentorship-backend)
```
Comment (satya7289) [inline README.md:4]: Now you have added the coverage report badge for mentorship-backend. Please update it for VMS.

Comment (aniketrb-github) [reply README.md:4]: Oh. I pasted it thinking, it was a common dashboard for different projects.

Is this link the correct one:
`link: https://codecov.io/gh/anitab-org/vms` 
`image: https://codecov.io/gh/anitab-org/vms/branch/develop/graph/badge.svg`
Can you please confirm.

**Review by satya7289 — APPROVED** [review]
Looks great now. 
One last thing you should do, squash the commits to one commit.

**Comment by aniketrb-github** [comment]
> Looks great now.
> One last thing you should do, squash the commits to one commit.

Hi @satya7289, I have squashed the commits into a single commit. Can you please review it. Thanks.

**Comment by satya7289** [comment]
> > Looks great now.
> > One last thing you should do, squash the commits to one commit.
> 
> Hi @satya7289, I have squashed the commits into a single commit. Can you please review it. Thanks.

I see there are 5 commits in this PR. Please squash it to 1 commit.

**Comment by aniketrb-github** [comment]
> > > Looks great now.
> > > One last thing you should do, squash the commits to one commit.
> > 
> > 
> > Hi @satya7289, I have squashed the commits into a single commit. Can you please review it. Thanks.
> 
> I see there are 5 commits in this PR. Please squash it to 1 commit. If you facing any difficulty in squashing then please let me know.

I did this squash commit action on my 3 distinct commits into a single commit for the first time yesterday & it was successful. This is the [new squashed commit](https://github.com/anitab-org/vms/pull/1064/commits/4cd154367fcc602cfe1b902214a49c6b90675539) that is formed after the squash action performed by me via gitbash. I used the "s"/"squash" option instead of "f"/"fix". Attaching the screenshot. 
squash-ss
So I will try to squash & use the "f"/"fix" option this time & let you know if I need help.

**Comment by aniketrb-github** [comment]
I have done the commit squash activity once again, just new with this activity.
Can you please check [my branch's commit history](https://github.com/aniketrb-github/vms/commits/vms-1062) once?
And this is my [child branch](https://github.com/aniketrb-github/vms/tree/vms-1062). Please let me know If this suffices? Thanks

**Review by satya7289 — APPROVED** [review]
Yes, This looks great. 
Thank you for submitting this PR.
You should take other issues as well for successful `Open Source Hack`, Happy coding..!

**Comment by aniketrb-github** [comment]
> Yes, This looks great. 
> Thank you for submitting this PR.
> You should take other issues as well for successful `Open Source Hack`, Happy coding..!
> 

Thanks @satya7289. You've been patient and helpful with a first timer like me. Sure I'm looking forward to submit more PRs.

**Comment by aniketrb-github** [comment]
@satya7289 
1) Just out of curiosity, basically how many reviews are necessary for a PR to be seen fit to be merged in the master/main branch?
And while the review process is onn, I can work on other issues in parallel right?

**Comment by satya7289** [comment]
> @satya7289 
> 1) Just out of curiosity, basically how many reviews are necessary for a PR to be seen fit to be merged in the master/main branch?
> 2) And while the review process is onn, I can work on other issues in parallel right?

1> min 2.
2>yes, please. You can work on others issue in parallel.

**Comment by aniketrb-github** [comment]
> > @satya7289
> > 
> > 1. Just out of curiosity, basically how many reviews are necessary for a PR to be seen fit to be merged in the master/main branch?
> > 2. And while the review process is on, I can work on other issues in parallel right?
> 
> 1> min 2.
> 2>yes, please. You can work on others issue in parallel.

oh ok, got it. thanks.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-AnitaB.org Portal - VMS project
+Volunteer Management System (VMS)
```
Comment (gaurivn) [inline README.md:1]:
```suggestion
AnitaB.org Portal - VMS project
```
I would suggest to undo the change made here since this is not included as part of the issue.

**Review by gaurivn — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (satya7289) [reply README.md:1]: @gaurivn I had updated the issue regarding this as well.

**Review by gaurivn — APPROVED** [review]

**Comment by gaurivn** [comment]
@satya7289, Idk it says I deleted, was trying to delete my previous review. 

@aniketrb-github, thank you for your contribution :clap:

**Review by isabelcosta — APPROVED** [review]

**Comment by aniketrb-github** [comment]
Hi @isabelcosta, I have a small query regarding this commit merge, just trying to understand the phase/cycle here.
Once a commit is merged, should we expect to view those code changes on the `develop` branch immediately?

If YES, I just checked the `develop` branch but I didn't see the **codecov badge** in the README.MD file for which I had made the changes.

If NO, then we do the code changes get integrated & start reflecting in the `develop` branch.

**Comment by isabelcosta** [comment]
@aniketrb-github you should be able to see it on the README right away. I think it got mistakenly deleted in this commit: https://github.com/anitab-org/vms/commit/bd155a6ee3fbaa18964bd85c2d6a14829a74fc8c

Could you submit a PR again with the badge? I am sorry we didn't notice this with the other PR https://github.com/anitab-org/vms/pull/1107

Thank you for bringing that up, we should pay more attention next time 🙈

**Comment by aniketrb-github** [comment]
> @aniketrb-github you should be able to see it on the README right away. I think it got mistakenly deleted in this commit: [bd155a6](https://github.com/anitab-org/vms/commit/bd155a6ee3fbaa18964bd85c2d6a14829a74fc8c)
> 
> Could you submit a PR again with the badge? I am sorry we didn't notice this with the other PR #1107
> 
> Thank you for bringing that up, we should pay more attention next time

Can we reuse or reopen this PR and use it again? 
If not possible, I will create a new PR.
I had forked develop branch, and not its a stale fork wherein I have solved 2 issues, so just asking about the reuse.

**Comment by isabelcosta** [comment]
@aniketrb-github its not possible to re open, you have to open another PR :/
You can create a new branch, and first update your develop branch. Do you know how to do that?

**Comment by aniketrb-github** [comment]
> @aniketrb-github it's not possible to reopen, you have to open another PR :/
> You can create a new branch, and first update your develop branch. Do you know how to do that?

Ok fine no problem, I will submit a new PR. Thanks.
